cfct/template/vpc_flowlog_logging_bucket.template

```diff
@@ -0,0 +1,116 @@
+# This is a sample, non-production-ready template
+# © 2021 Amazon Web Services, Inc. or its affiliates. All Rights Reserved.  
+# This AWS Content is provided subject to the terms of the AWS Customer Agreement available at  
+# http://aws.amazon.com/agreement or other written agreement between Customer and either
+# Amazon Web Services, Inc. or Amazon Web Services EMEA SARL or both.
+
+AWSTemplateFormatVersion: 2010-09-09
+Description: Create a S3 storage bucket in the audit account.
+
+Parameters:
+  OrganizationId:
+    Type: 'String'
+    Description: ID of the AWS Organization of the Landing Zone
+  SSEAlgorithm:
+    Type: 'String'
+    Default: 'AES256'
+    Description: S3 bucket SSE Algorithm.
+    AllowedValues:
+    - 'AES256'
+    - 'aws:kms'
+  KMSMasterKeyID:
+    Type: 'String'
+    Description: 'KMS key ID required if SSE algorithm is aws:kms.'
+
+Conditions:
+  UseKMS: !Equals
+    - !Ref SSEAlgorithm
+    - 'aws:kms'
+  UseAES256: !Equals
+    - !Ref SSEAlgorithm
+    - 'AES256'
+
+Resources:
+
+  S3BucketFlowLoggingSSE:
+    Type: AWS::S3::Bucket
+    Condition: UseAES256
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      BucketName:  !Sub control-tower-vpcflowlogs-${AWS::AccountId}-${AWS::Region}
+      BucketEncryption:
+        ServerSideEncryptionConfiguration: 
+          - ServerSideEncryptionByDefault: 
+              SSEAlgorithm: AES256
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W35
+            reason: "Supress bucket access logging requirement"
+
+  S3BucketFlowLoggingKMS:
+    Type: AWS::S3::Bucket
+    Condition: UseKMS
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      BucketName:  !Sub control-tower-vpcflowlogs-${AWS::AccountId}-${AWS::Region}
+      BucketEncryption:
+        ServerSideEncryptionConfiguration: 
+          - ServerSideEncryptionByDefault:
+              KMSMasterKeyID: !Ref KMSMasterKeyID
+              SSEAlgorithm: !Ref SSEAlgorithm
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W35
+            reason: "Supress bucket access logging requirement"
+
+  BucketPolicy:

```

Comment:
can you add 1 extra policy to enforce secure SSL as shown in [c9a6e3e](https://github.com/aws-samples/aws-ct-vpc-flowlogs/pull/6/commits/a52873a911d390010d47c761233df3fe4228dfd9)

Reply:
done